test/test_datasets.py

```diff
@@ -1702,5 +1702,43 @@ def test_classes(self, config):
             self.assertSequenceEqual(dataset.classes, info["classes"])
 
 
+class KittiTestCase(datasets_utils.ImageDatasetTestCase):
+    DATASET_CLASS = datasets.Kitti
+    FEATURE_TYPES = (PIL.Image.Image, (list, type(None)))  # test split returns None as target
+    ADDITIONAL_CONFIGS = datasets_utils.combinations_grid(split=("train", "test"))
+
+    def inject_fake_data(self, tmpdir, config):
+        kitti_dir = os.path.join(tmpdir, "Kitti", "raw")
+        os.makedirs(kitti_dir)
+
+        split_to_idx = split_to_num_examples = {
+            None: 1,
+            "train": 1,
+            "test": 2,
+        }
+
+        # We need to create all folders regardless of the split in config
+        for split in ("train", "test"):
```

Comment:
This kinda confirms my suspicion from above. While we have  the ability to pass `split=None` it is not even used in our tests. Thus, I suggest we remove that.